src/Migrator.php

```diff
@@ -110,7 +114,14 @@ public function processRequestMigrations() : Request
             })
             ->flatten()
             ->each(function ($migration) {
-                $this->request = (new $migration())->migrateRequest($this->request);
+                $class = (new $migration());
+                $originalRequest = $this->request;
+
+                event(new RequestIsMigrating($class, $originalRequest));
```

Comment:
Maybe use the Facade rather than the helper since that's the pattern we've been using.